skyportal/handlers/api/followup_request.py

```diff
@@ -1027,3 +1033,182 @@ async def get(self, instrument_id):
         data = io.BytesIO(rez["data"])
 
         await self.send_file(data, filename, output_type=output_format)
+
+
+class FollowupRequestPrioritizationHandler(BaseHandler):
+    @auth_or_token
+    async def put(self):
+        """
+        ---
+        description: Reprioritize followup requests schedule
+        tags:
+            - followup_requests
+        parameters:
+        - in: query
+          name: observationStartDate
+          nullable: true
+          schema:
+            type: string
+          description: |
+            Arrow-parseable date string (e.g. 2020-01-01). If provided, start time
+            of observation window, otherwise now.
+        - in: query
+          name: observationEndDate
+          nullable: true
+          schema:
+            type: string
+          description: |
+            Arrow-parseable date string (e.g. 2020-01-01). If provided, end time
+            of observation window, otherwise 12 hours from now.
+        - in: query
+          name: instrumentId
+          schema:
+            type: string
+          description: Filter by instrument ID
+        - in: query
+          name: localizationId
+          schema:
+            type: integer
+          description: Filter by localization ID
+        - in: query
+          name: requestIds
```

Comment:
Usually when you use PUT, it comes with a single path argument that identifies the thing you're updating. Here we specify localization and instrument IDs (one of each, I guess) and then a bunch of followup IDs. Maybe we should instead do this for each followup request in a separate call? 
Or if that is not efficient, and you still want to give a list of followup IDs, do you actually need the instrument and localization IDs, or can you grab them from the followup requests themselves?

Reply:
Yeah maybe we have a chat about this.